### PR TITLE
Send to nos account moderation

### PR DESCRIPTION
**Pull request by dcadenas**
## Issues covered
Covers https://github.com/planetary-social/nos/issues/1083, https://github.com/planetary-social/nos/issues/1093, https://github.com/planetary-social/cleanstr/issues/44, https://github.com/planetary-social/nos/issues/1084
## Description

It adds a `Send To Nos` option in the account menu similar to the note version so that users can have flag anonymously

## How to test
1. Navigate to some user's menu
2. Select Flag this User
3. Select a category
4. Select `Send to Nos`
5. Confirm Dialog
6. Verify report in Slack channel

### CODE REVIEW

**Review by mplorentz — CHANGES_REQUESTED** [review]
I tested all the reporting flows: public note, anonymous note, public author, anonymous author. All look good to me! Nice work!

However there were some configuration changes we'll need to revert before merging.

File: Nos.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2320,7 +2320,7 @@
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Nos/Views/Preview Content\"";
 				DEVELOPMENT_TEAM = "";
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = GZCZBKH7MY;
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = FU5BU675QQ;
```
[the remark this answers is not in the document]
Comment (dcadenas) [reply Nos.xcodeproj/project.pbxproj:2323]: I think I was trying to run the app in the real phone and I was using my ID but I don't really know how to and my account is not up to date. Not a big deal really, I don't need it

**Review by mplorentz — APPROVED** [review]
Ready to merge!

**Comment by joshuatbrown** [comment]
I'll merge since I think Daniel is still out and I'm working on a related ticket.

**Pull request merged**